Update CONTRIBUTING.md

mallenexpensify opened the pull request:
Added - `Please use the templates for bugs and feature requests that are bookmarked in #expensify-open-source`.  
Removed text about inc. details cuz those are in the template.

<!-- If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit. -->

### Details
<!-- Explanation of the change or anything fishy that is going on -->
Added - `Please use the templates for bugs and feature requests that are bookmarked in #expensify-open-source`.  
Removed text about inc. details cuz those are in the template.


### Fixed Issues
<!---
Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing.
Do NOT add the special GH keywords like `fixed` etc, we have our own process of managing the flow.
It MUST be an entire link to the issue; otherwise, the linking will not work as expected.

Make sure this section looks similar to this (you can link multiple issues using the same formatting, just add a new line):

$ https://github.com/Expensify/App/issues/<number-of-the-issue>

Do NOT only link the issue number like this: $ #<number-of-the-issue>
--->
$ GH_LINK

### Tests
<!---
Add a numbered list of manual tests you performed that validates your changes work on all platforms, and that there are no regressions present.
Add any additional test steps if test steps are unique to a particular platform.
Manual test steps should be written so that your reviewer can repeat and verify one or more expected outcomes in the development environment.

For example:
1. Click on the text input to bring it into focus
2. Upload an image via copy paste
3. Verify a modal appears displaying a preview of that image
--->
Ensure text reads `Please use the templates for bugs and feature requests that are bookmarked in #expensify-open-source`.  

- [ ] Verify that no errors appear in the JS console

### QA Steps
<!---
Add a numbered list of manual tests that can be performed by our QA engineers on the staging environment to validate that your changes work on all platforms, and that there are no regressions present.
Add any additional QA steps if test steps are unique to a particular platform.
Manual test steps should be written so that the QA engineer can repeat and verify one or more expected outcomes in the staging environment.

For example:
1. Click on the text input to bring it into focus
2. Upload an image via copy paste
3. Verify a modal appears displaying a preview of that image
--->

- [ ] Verify that no errors appear in the JS console

### Tested On

- [ ] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### Screenshots
<!-- Add screenshots for all platforms tested. Pull requests won't be merged unless the screenshots show the app was tested on all platforms.-->

#### Web
<!-- Insert screenshots of your changes on the web platform-->

#### Mobile Web
<!-- Insert screenshots of your changes on the web platform (from a mobile browser)-->

#### Desktop
<!-- Insert screenshots of your changes on the desktop platform-->

#### iOS
<!-- Insert screenshots of your changes on the iOS platform-->

#### Android
<!-- Insert screenshots of your changes on the Android platform-->

mallenexpensify committed:
Update CONTRIBUTING.md

Added - Please use the templates for bugs and feature requests that are bookmarked in #expensify-open-source.  
Removed text about inc. details cuz those are in the template.

mountiny reviewed (commented):
Should we maybe include a link to the How to create a plan instruction somewhere in this file too?

mallenexpensify commented:
Yes! Good call @mountiny , can you see where it'd fit best and add it? (it'll be quicker for you, I imagine)

mountiny commented on CONTRIBUTING.md:
```diff
@@ -55,7 +55,7 @@ Please follow these steps to propose a job:
 4. If there is no existing GitHub issue or Upwork job, check if the issue is happening on prod (as opposed to only happening on dev)
 5. If the issue is just in dev then it means it's a new issue and has not been deployed to production. In this case, you should try to find the offending PR and comment in the issue tied to the PR and ask the assigned users to add the `DeployBlockerCash` label. If you can't find it, follow the reporting instructions in the next item, but note that the issue is a regression only found in dev and n ...
-6. If the issue happens in production then report the issue(s) in the [#expensify-open-source](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#asking-questions) Slack channel, prefixed with `BUG:` or `Feature Request:`. Include all relevant data, screenshots and examples in the post or thread.
+6. If the issue happens in production then report the issue(s) in the [#expensify-open-source](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#asking-questions) Slack channel, prefixed with `BUG:` or `Feature Request:`. Please use the templates for bugs and feature requests that are bookmarked in #expensify-open-source.  
```
@mallenexpensify To be honest, I believe this line is the best in this file. I feel like the instructions are most handy when someone wants to propose a new feature, which is mentioned on this line. So I suggest:

```suggestion
6. If the issue happens in production then report the issue(s) in the [#expensify-open-source](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#asking-questions) Slack channel, prefixed with `BUG:` or `Feature Request:`. Please use the templates for bugs and feature requests that are bookmarked in #expensify-open-source.  For instructions on how to come up with a plan, which could be useful for you in this stage especially in case of a feature request, please refer to [this guide](https://github.com/Expensify/App/blob/main/HOW_TO_CREATE_A_PLAN.md).
```

mallenexpensify replied:
@mountiny , this shortened version will likely work better. 
> View [this guide](https://github.com/Expensify/App/blob/main/HOW_TO_CREATE_A_PLAN.md) for help creating a plan when proposing a feature request

How would I delete yours and add mine above? I'm unfamiliar with editing a PR

mountiny replied:
@mallenexpensify I think what you can do is to click on the branch name here:
![image](https://user-images.githubusercontent.com/36083550/152010188-b16c62fc-162d-467f-ac80-a5749be839fc.png)

`mallenexpensify-patch-1` in this case. That will bring you to your version of the code which we can see in this PR. Then navigate to the Contributing file https://github.com/Expensify/App/blob/mallenexpensify-patch-1/CONTRIBUTING.md

And then you cum click the pencil button same as before, make the changes and commit it.

mountiny reviewed (commented):
Left a suggestion there, feel free to adjust it as you find best.

mallenexpensify commented:
Thanks @mountiny , that gets me to the edit page but I don't see your proposed changes there to edit
![image](https://user-images.githubusercontent.com/22508304/152011992-93fb13cc-d8be-46ad-b3d9-3cfb967e9ca9.png)

Would I just make the intended final edits there then Github would use 'magic' to decipher the differences between your recommended updates and mine?

mountiny commented:
@mallenexpensify That is correct, what you will see there are only the changes you have made. You can then copy this line and put it in there and in the bottom add commit and it will show up here!

```
View [this guide](https://github.com/Expensify/App/blob/main/HOW_TO_CREATE_A_PLAN.md) for help creating a plan when proposing a feature request.
```

mallenexpensify committed:
Update CONTRIBUTING.md

> View [this guide](https://github.com/Expensify/App/blob/main/HOW_TO_CREATE_A_PLAN.md) for help creating a plan when proposing a feature request.

mallenexpensify commented:
I did what you recommended above @mountiny , should it show up here now?

mountiny reviewed (approved):
Yes! Amazing! You can see in the top header menu in the PR that there are 2 commits.
![image](https://user-images.githubusercontent.com/36083550/152014754-3b82d220-1ad9-4e3a-a4be-9babd6fe8281.png)
Jut gonna wait for the tests.

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/1779856312) to staging by @mountiny in version: 1.1.33-5 🚀

 platform | result 
 ---|--- 
🤖 android 🤖|success ✅ 
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|failure ❌ 
🕸 web 🕸|success ✅

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/1785535610) to production by @sketchydroide in version: 1.1.34-0 🚀

 platform | result 
 ---|--- 
🤖 android 🤖|success ✅ 
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|success ✅ 
🕸 web 🕸|success ✅
